src/core/flow.js

```diff
@@ -471,10 +478,6 @@ export function FlowWithPhases({
       state = dispatch(state, { type: 'endPhase', args: [end], playerID: action.playerID });
```

Comment:
I wasn't sure if (for the sake of consistency) we should end the phase too... Let me know if you think it is a good idea, I can change the code.